spec/support/dependency_helpers.rb

```diff
@@ -23,23 +24,22 @@ def build_gem(gem_name, version = "1.0.0")
               s.homepage = 'http://github.com/thoughtbot/#{gem_name}'
               s.required_ruby_version = '>= 2.3.0'
             end
-          GEMSPEC
-        end
+        GEMSPEC
+      end
 
-        File.open lib_file, "w" do |file|
-          file.puts "$#{gem_name}_version = '#{version}'"
-        end
+      File.open lib_file, "w" do |file|
+        file.puts "$#{gem_name}_version = '#{version}'"
+      end
 
-        redirect = ENV["VERBOSE"] ? "" : "2>&1"
+      redirect = ENV["VERBOSE"] ? "" : "2>&1"
 
-        puts "building gem: #{gem_name} #{version}" if ENV["VERBOSE"]
-        `gem build #{gemspec} #{redirect}`
+      puts "building gem: #{gem_name} #{version}" if ENV["VERBOSE"]
+      %x[gem build #{gemspec} #{redirect}]
```

Comment:
Can you please use `%x()`? It reads better than `%x[]` or `%x{}`. Thank you!

Reply:
Fixed in #250